### PR TITLE
Draft: Jared's first branch

### PR DESCRIPTION
**Pull request by JaredTheHammer**
Test first pull request

### CODE REVIEW

File: client/src/assets/specialists/Space_ship_corvette.svg
Comment (GavPierce) [inline client/src/assets/specialists/Space_ship_corvette.svg:1]: This seems to be the same as the other space-ship.
Also, when we implement Specialists pointing to direction of travel, we'll need them all to be uniform, like all pointing up by default.

**Comment by GavPierce** [comment]
@JaredTheHammer It seems you're still adding things to this PR even after we've approved it, let me know when you are done and I will review the new changes.

File: client/src/assets/specialists/Space_ship_corvette.svg
Comment (JaredTheHammer) [reply client/src/assets/specialists/Space_ship_corvette.svg:1]: I agree

**Comment by JaredTheHammer** [comment]
> @JaredTheHammer It seems you're still adding things to this PR even after we've approved it, let me know when you are done and I will review the new changes.

I'm sorry I still am learning how this flows I need to make a new branch after submitting a pull request correct?

**Comment by JaredTheHammer** [comment]
@GavPierce 
I won't make any more changes to the code until you review it and merge it with the main branch.

**Review by GavPierce — APPROVED** [review]
Looks good! Headed to bed so I will leave it to your discretion to fix the svgs now or later.

**Pull request merged**
